src/envoy/http/mixer/report_data.h

```diff
@@ -45,15 +45,17 @@ class ReportData : public ::istio::control::http::ReportData {
   }
 
   void GetReportInfo(
-      ::istio::control::http::ReportData::ReportInfo* data) const override {
+      ::istio::control::http::ReportData::ReportInfo *data) const override {
     data->received_bytes = info_.bytesReceived();
     data->send_bytes = info_.bytesSent();
-    data->duration =
-        std::chrono::duration_cast<std::chrono::nanoseconds>(info_.duration());
+    data->duration = std::chrono::nanoseconds{0};
```

Comment:
Use value_or() instead of these 4 lines.

Reply:
Done.